wagtailio/static/sass/components/_text-media.scss

```diff
@@ -0,0 +1,72 @@
+.text-media {
+    $root: &;
+
+    &__content {
+        margin: auto;
+        grid-row: 2;
+        grid-column: 2 /span 2;
+        padding: 10px 10px 0 0;
+
+        @include media-query(medium) {
+            grid-column: 2/ span 2;
```

Comment:
suggestion: remove this line as it's already set on line 7

Reply:
Good point!